genericapi/handler.go

```diff
@@ -150,5 +164,12 @@ func (h *Handler) ServeCreateAlert(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	w.WriteHeader(204)
+	if r.Header.Get("Accept") != "application/json" {
+		w.WriteHeader(204)
+		return
+	}
+
+	data, _ := json.Marshal(&resp)
+	_, err = w.Write(data)
+	errutil.HTTPError(ctx, w, errors.Wrap(err, "create alert send payload"))
```

Comment:
This can be ignored, since the remote side has presumably already terminated the connection
```suggestion
	_,_ = w.Write(data)
```